src/patternfly/components/DataList/data-list.scss

```diff
@@ -176,6 +182,30 @@
       font-size: var(--pf-c-data-list--m-compact__check--FontSize);
     }
   }
+
+  &.pf-m-truncate {
+    .pf-c-data-list__cell,
+    .pf-c-data-list__text {
+      --pf-c-data-list__cell--MaxWidth: var(--pf-c-data-list__cell--m-truncate--MaxWidth);
+      --pf-c-data-list__cell--Overflow: hidden;
+      --pf-c-data-list__cell--TextOverflow: ellipsis;
+      --pf-c-data-list__cell--WhiteSpace: nowrap;
+    }
+  }
+
+  &.pf-m-break-word {
+    .pf-c-data-list__cell,
+    .pf-c-data-list__text {
+      --pf-c-data-list__cell--WordBreak: break-word;
```

Comment:
We should probably model this off of the table and add `--pf-c-data-list__cell--WhiteSpace: normal;` so that `pf-m-break-word` can be added to a cell in a table that has `.pf-m-nowrap`, and it will enable that cell to wrap. We could also add a `.pf-m-wrap` modifier to do the same thing. Looks like that's missing in the data-list

Reply:
Ah yes good point. I didn't add pf-m-wrap because the data list cells wrap by default. Should I still add it?